projects.md

```diff
@@ -0,0 +1,45 @@
+
+Members leading a sales effort should draft an **internal proposal** and present it at a meeting to see whether the co-operative wants to move forward to submit a proposal, whether there are concerns by other members, and gauge team capacity given project timelines. A sample internal proposal should include:
+
+```
+# Project title
+
+## Background
+Background information about the client/grant and the project.
+
+## Scope & timeline
+What are the expected deliverables and the timeline.
+
+## Outcomes & opportunities
+How this project may be beneficial beyond the scope of the project. For example, does it lead to developing new skillsets among members or starting a relationship with a potentially long-term client?
+
+## Skillsets
+What are the important skillsets needed to ship this. If some skillsets are not available, what are the plans to source them to ensure we can deliver on this?
+
+## Budget
+Project budget and payment terms if available.
+
+## Communication
+How the logistics would look like for members working on this project. Are there weekly calls or are there physical sessions? Is the project on GitHub or is the source closed? Are there consent and confidentiality terms?
+
+## Risks
+What are the expected risks associated with this project.
+
+## Resources
+Additional resources about the project not already listed above, such as link to a website or a call for proposal.
+```
+
+You can add more sections as appropriate for the project. This may include sample code, project roadmap, next steps, etc. You can find examples in our archived proposals folder in our [shared drive](http://link.hypha.coop/drive).
+
+## Proposal drafting and submissions
+
+Project proposals to be submitted to a potential client or grant committee should be drafted by members who expect to participate in the project with the help of members of the Business Planning WG. The WG has expertise in drafting client and grant project proposals, and managing the proposal process, so it is helpful to involve members of that group from the drafting stage. All that are involved in the proposal drafting make up the **proposal committee** that will eventually sign off on the final draft to be submitted.
+
+If a project is accepted, this committee will lead the engagement meetings with the client (e.g. discuss terms of payment and project logistics) and gradually transfer the relationship entirely to the project team, which may not necessarily include everyone in the proposal committee.
+
+Regardless whether the proposal is successful, we should snapshot artifacts in the archived proposals folder for future reference. For failed proposals, we highly encourage reaching out the the client or grant committee for feedback and have an internal retrospective.
```

Comment:
💯 to followup on failures. Would even go as far as to say "always" -- no reason not to learn :)

Reply:
Thoughts on my stubbing out a guide for this? (I did a bit of research, and felt like I learned some decent tactics)

Reply:
I think _always_ is unnecessarily restrictive. There are external factors that may lead us to not want to follow up. For example, prioritization of existing tasks.

I don't feel we need a fixed guide for this. The strategy will keep changing and fixing a document is too rigid and may get out of date. Don't want to get into a case where the practice is no longer in sync with doc.